eslint.config.mjs

```diff
@@ -0,0 +1,59 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+import { includeIgnoreFile } from "@eslint/compat";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+const gitignorePath = path.resolve(__dirname, ".gitignore");
+
+export default [
+    includeIgnoreFile(gitignorePath),
+    ...compat.extends("brightspace/lit-config").map(c => ({
+		...c,
+		files:['**/*.js','**/*.html']
+	})),
+    {
+        languageOptions: {
+            globals: {
+                Prism: false,
+            },
+        },
+    },
+    {
+        files: ['**/*.js','**/*.html'],
+        rules: {
+            "no-restricted-syntax": [0, "CatchClause[param=null]"]
+        },
+    },
+    {
+        files: ["**/demo/**.html"],
```

Comment:
Not for this PR, but maybe we should add this to the shared `lit-config`? Allow consoles in demos?

Reply:
That makes sense, I can also take a look at other configs to see if they are making the same override and remove those as well.